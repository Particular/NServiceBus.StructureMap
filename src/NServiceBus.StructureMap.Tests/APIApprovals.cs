using NServiceBus.StructureMap;
using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(StructureMapObjectBuilder).Assembly, excludeAttributes: new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" });
        Approver.Verify(publicApi);
    }
}
