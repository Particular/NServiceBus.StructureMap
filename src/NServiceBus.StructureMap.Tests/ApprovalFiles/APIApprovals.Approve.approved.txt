[assembly: System.CLSCompliantAttribute(true)]
[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("NServiceBus.StructureMap.Tests")]
[assembly: System.Runtime.InteropServices.ComVisibleAttribute(false)]
namespace NServiceBus
{
    [System.ObsoleteAttribute(@"StructureMap is no longer supported via the NServiceBus.StructureMap adapter. NServiceBus directly supports all containers compatible with Microsoft.Extensions.DependencyInjection.Abstractions via the externally managed container mode. Will be removed in version 9.0.0.", true)]
    public class StructureMapBuilder
    {
        public StructureMapBuilder() { }
    }
    [System.ObsoleteAttribute(@"StructureMap is no longer supported via the NServiceBus.StructureMap adapter. NServiceBus directly supports all containers compatible with Microsoft.Extensions.DependencyInjection.Abstractions via the externally managed container mode. Will be removed in version 9.0.0.", true)]
    public class static StructureMapExtensions { }
}